GUI/Src/main.py

```diff
@@ -52,26 +53,33 @@ def __init__(self):
 
         #Place the navigation, IMU, orientation, and odometery display widget
         self.set_nav_odometery()
-        self.set_pid_visualizer()
+        #self.set_pid_visualizer()
```

Comment:
Was this intentionally commented out? I think this may have been left in here for debug testing

Reply:
Whoops that was a mistake. Will fix